pkg/resource/models/service_resource.go

```diff
@@ -98,6 +98,7 @@ func (b BackupConfiguration) ObjectValue() basetypes.ObjectValue {
 
 type ServiceResourceModel struct {
 	ID                              types.String `tfsdk:"id"`
+	BYOCId                          types.String `tfsdk:"byoc_id"`
```

Comment:
```suggestion
	BYOCID                          types.String `tfsdk:"byoc_id"`
```

In same struct, there is `ID` field, IMHO makes sense to follow same style for other fields. (and for models.Service struct, but there is `Id` field).

Reply:
fair point